dvc/command/plot.py

```diff
@@ -148,7 +148,7 @@ def add_parser(subparsers, parent_parser):
         "--stdout",
         action="store_true",
         default=False,
-        help="Print plot specification to stdout.",
+        help="Print plot location to stdout.",
```

Comment:
So this option lets user print the plot as a text to stdout. So, plot by default logs plot location to stdout. When using `--stdout` option, it is the plot file content that will be displayed on stdout.

Reply:
Oh I see. Got it wrong then, it seems. Let me FTFY ⏳